src/PensionsRegulatorApi/Application/Queries/GetOrganisationsHandler.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using MediatR;
+using PensionsRegulatorApi.Data;
+using PensionsRegulatorApi.Domain;
+
+namespace PensionsRegulatorApi.Application.Queries
+{
+    
+
+    public class GetOrganisationsHandler : RequestHandler<GetOrganisations, IEnumerable<Organisation>>
+    {
+        private readonly OrganisationRepository _repository;
+
+        public GetOrganisationsHandler(OrganisationRepository repository)
+        {
+            _repository = repository ?? throw new ArgumentNullException(nameof(repository));
```

Comment:
Is this a pattern we follow elsewhere in constructor injection?

Reply:
Seems to vary depending on repo. I called it out as not bein used in a code review some time back :)

There seems to be a view that IoC will deal with it anyway. Alternative view is with this pattern still works without IoC i.e. "poor man's DI" 

I can remove if is an issue.